volplugin/handlers.go

```diff
@@ -226,6 +227,7 @@ func mount(master, host string, ttl int) func(http.ResponseWriter, *http.Request
 
 		actualSize, err := volConfig.Options.ActualSize()
 		if err != nil {
+			removeStopChan(uc.Request.Name)
```

Comment:
The problem with this is that code expects err to be global scope, I really bit myself trying to do it this way last time in some of the rbd locking code. There's really no good pattern in go for this, I think. I mean, if you think you can get it right and not make the code harder to read, I'd love a patch, but I don't think I can do it.